mimic/test/test_loadbalancer.py

```diff
@@ -812,6 +812,32 @@ def test_add_multiple_nodes(self):
         self.assertEqual(create_node_response.code, 202)
         self.assertEqual(len(create_node_response_body["nodes"]), 2)
 
+    def test_add_multiple_nodes_feeds(self):
+        """
+        Ensure there are "Node created" feed for each node after creating nodes
+        """
+        node_ids = [
+            body["nodes"][0]["id"]
+            for resp, body in self._create_nodes(["1.1.1.1", "1.1.1.2"])]
+        node_feeds = [
+            ("<feed xmlns=\"http://www.w3.org/2005/Atom\"><entry>"
+             "<summary>Node successfully created with address: '1.1.1.1', "
+             "port: '80', condition: 'ENABLED', weight: '10'</summary>"
+             "<updated>1970-01-01T00:00:00.000000Z</updated></entry></feed>"),
+            ("<feed xmlns=\"http://www.w3.org/2005/Atom\"><entry>"
+             "<summary>Node successfully created with address: '1.1.1.2', "
+             "port: '80', condition: 'ENABLED', weight: '10'</summary>"
+             "<updated>1970-01-01T00:00:00.000000Z</updated></entry></feed>")
+        ]
+        for node_id, exp_node_feed in zip(node_ids, node_feeds):
```

Comment:
I'd really like to get away from using loops in tests.  Could you express this as a few repeated calls to a function, instead, so that the traceback encodes information about what case is failing, by the line number that has failed?